docs/documentation/pages/DECKHOUSE_CONFIGURE.md

```diff
@@ -178,6 +178,9 @@ You cannot set `nodeSelector` and `tolerations` for modules:
     * It checks if a node with the <code>node-role.deckhouse.io/MODULE_NAME</code> label is present in the cluster.
     * It checks if a node with the <code>node-role.deckhouse.io/monitoring</code> label is present in the cluster.
     * It checks if a node with the <code>node-role.deckhouse.io/system</code> label is present in the cluster.
+  * Deckhouse examines nodes to determine a longtermNodeSelector in the following order:
+    * It checks if a node with the <code>node-role.deckhouse.io/monitoring-longterm</code> label is present in the cluster.
+    * Otherwise, the logic for the nodeSelector section will be executed.
```

Comment:
```suggestion
  * Deckhouse examines nodes to determine a [longtermNodeSelector](modules/300-prometheus/configuration.html#parameters-longtermretentiondays) in the following order:
    * It checks if a node with the `node-role.deckhouse.io/monitoring-longterm` label is present in the cluster.
    * Otherwise, the logic for the [nodeSelector](modules/300-prometheus/configuration.html#parameters-nodeselector) section will be executed.
```